lib/checks/mobile/target-offset.json

```diff
@@ -8,7 +8,11 @@
     "impact": "serious",
     "messages": {
       "pass": "Target has sufficient offset from its closest neighbor (${data.closestOffset}px should be at least ${data.minOffset}px)",
-      "fail": "Target has insufficient offset from its closest neighbor (${data.closestOffset}px should be at least ${data.minOffset}px)"
+      "fail": "Target has insufficient offset from its closest neighbor (${data.closestOffset}px should be at least ${data.minOffset}px)",
+      "incomplete": {
+        "default": "Potential target has insufficient offset from its closest neighbor (${data.closestOffset}px should be at least ${data.minOffset}px)",
+        "nonTabbableNeighbor": "Target has insufficient offset from a potential other target (${data.closestOffset}px should be at least ${data.minOffset}px)"
```

Comment:
I'm sort of anticipating that inert will be a thing soon too.

Reply:
Looks like you don't have to wait long, it's now available in all browsers except Firefox, but is available in a nightly build. https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert#browser_compatibility. In that case, we should say something about how the element is not focusable instead of `tabindex=-1`.

Reply:
On second thought, `inert` would make an element inapplicable in this rule because it's not focusable. I made some changes. See what you think.